pkg/metrics_pusher/pusher.go

```
@@ -62,23 +62,24 @@ func (p *MetricsPusher) pushMetrics() {
 			return
 		}
 	} else {
+		var resp *http.Response
 		caCert, err := ioutil.ReadFile(p.Clientca)
 		if err != nil {
-			p.Log.Error(err, "Unable to read CA cert for fetching metrics: Please provide valid CA cert or path")
-			return
+			p.Log.Info("Unable to read CA cert for fetching metrics, going to assuming none are needed.")
```

We should not assume that we can ignore errors if the file is not found.